x/stakeibc/keeper/hooks.go

```diff
@@ -41,18 +41,23 @@ func (k Keeper) BeforeEpochStart(ctx sdk.Context, epochInfo epochstypes.EpochInf
 	}
 	// deposit records *must* exist for this epoch
 	k.SetEpochTracker(ctx, epochTracker)
+	k.Logger(ctx).Info(fmt.Sprintf("Setting epochTracker %v", epochTracker))
```

Comment:
might be good to put this before line 43

Reply:
Done.